profiles/schema.py

```diff
@@ -0,0 +1,117 @@
+import graphene
+from django.conf import settings
+from django.db.models import CharField, Value
+from django.db.models.functions import Concat
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ("code",)
+
+    vocabulary = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+        fields = ("children", "origin_id", "ocd_id", "municipality")
+
+    type = graphene.String()
+    name = graphene.String()
+
+    def resolve_children(self, info, **kwargs):
+        return self.children.filter(type__type="sub_district")
+
+    def resolve_type(self, info, **kwargs):
+        return self.type.type
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+        fields = ("nickname", "image", "email", "phone")
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+    def resolve_concepts_of_interest(self, info, **kwargs):
+        return self.concepts_of_interest.all()
+
+    def resolve_divisions_of_interest(self, info, **kwargs):
+        return self.divisions_of_interest.all()
+
+
+Language = graphene.Enum("Language", [(l[1].upper(), l[0]) for l in settings.LANGUAGES])
```

Comment:
let's wrap these enums in:
```python
from django.utils import translation

...

with translation.override("en"):
    Language = graphene.Enum("Language", [(l[1].upper(), l[0]) for l in settings.LANGUAGES])
    ContactMethod = graphene.Enum(
        "ContactMethod", [(cm[1].upper(), cm[0]) for cm in settings.CONTACT_METHODS]
    )
```

just to make sure that if anyone turns their labels to translated strings, the error will not pop up